src/app/components/BibPage/BibPage.jsx

```diff
@@ -32,27 +32,24 @@ const BibPage = (props) => {
       <main className="main-page">
         <div className="nypl-page-header">
           <div className="nypl-full-width-wrapper">
-            <Breadcrumbs
-              query={searchURL.substring(2)}
-              type="bib"
-              title={title}
+            <h2>New York Public Library Research Catalog</h2>
+            <Search
+              searchKeywords={props.searchKeywords}
+              field={props.field}
+              spinning={props.spinning}
+              createAPIQuery={createAPIQuery}
             />
-            <h1>Research Catalog</h1>
-            <BackLink searchURL={searchURL} searchKeywords={props.searchKeywords} />
+            <div className="nypl-row search-control">
+              <svg xmlns="http://www.w3.org/2000/svg" width="25" height="42" viewBox="0 0 25 42" preserveAspectRatio="xMidYMid meet" aria-labelledby="left-arrow" aria-hidden="true">
```

Comment:
Is this because the Left Arrow is not suppose to be clickable? Can you reuse the SVG component instead of hardcoding the SVG here.